k8s/sumo/templates/deployment.yaml

```diff
@@ -0,0 +1,206 @@
+---
+# Web deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sumo.fullname" . }}-web
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "sumo.name" . }}-web
+    {{- include "sumo.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  revisionHistoryLimit: {{ .Values.config.web.revisionHistoryLimit }}
```

Comment:
I completely skipped the `strategy` section from here as I believe what we have currently it's the default

```yaml
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
```

Reply:
yeah, that's fine